source/puppet/4.9/style_guide.markdown

```diff
@@ -1090,7 +1090,7 @@ We recommend not mixing conditionals with resource declarations. When you use co
 
     file { '/tmp/readme.txt':
       ensure  => file,
-      content => "Hello World\n",
+      content => 'Hello World\n',
```

Comment:
This needs double quotes as it interpolates a `\n` (it would be a literal backslash-en inside single quotes).